lib/index.ts

```diff
@@ -0,0 +1,405 @@
+/**
+ * @packageDocumentation
+ * TopStats - Official Node.js client for the topstats.gg API
+ * @version 1.0.0
+ */
+
+/**
+ * Response structure for rate limit errors
+ */
+export interface RateLimitedResponse {
+  statusCode: number;
+  error: string;
+  message: string;
+  date: string;
+  expiresIn: string;
```

Comment:
This should be a number, docs incorrectly show this. Mentioned before.

Reply:
fixed